README.md

```diff
@@ -83,6 +83,12 @@ There are multiple files created and tested for different materials and scales.
 ```
 files/
 ├── 3mm_plywood
+|   └── domes
```

Comment:
Thanks for editing this as well, let's move it in the root folder like this

files > complete_models > domes > 4v_48cm_.... 

My files structure was not ready for contributions, but that's how it would be cool to keep it